app/controllers/home_controller.rb

```diff
@@ -1,5 +1,7 @@
 class HomeController < ApplicationController
   def index
     @latest_projects = Project.latest(5)
+    @latest_repositories = Repository.latest(5)
+    @latest_configurations = KalibroConfiguration.latest(5)
```

Comment:
Nice catch https://github.com/mezuro/prezento/issues/339 as this seems as a bug introduced back in 2013 and just found now.